distribution/changelog.txt

```diff
@@ -26,6 +26,7 @@
 - Fix: [#15213] Freeze when hovering over Reverse Freefall Coaster in Russian.
 - Fix: [#15255] Tile Inspector shows banner information on walls that do not contain one.
 - Fix: [#15257] Chat icon shows in scenario/track editor. Other icons don't disable when deactivated in options menu.
+- Fix: [#15274] Scenario description missing when loading a park from a save file.
```

Comment:
Since this bug is not present in the last release, this changelog entry is not needed.